src/masonite/orm/grammar/BaseGrammar.py

```diff
@@ -109,7 +109,7 @@ def _compile_alter_columns(self):
             sql += getattr(self, "{}_column_string".format(column._action))().format(
                 column=self._compile_column(column.column_name),
                 old_column=self._compile_column(column.old_column),
-                data_type=self.type_map.get(column.column_type),
+                data_type=self.type_map.get(column.column_type) if self.type_map.get(column.column_type) else "",
```

Comment:
You don't need this ternary operator here. the `.get()` on a dictionary takes a second argument for a default. You can just do `self.type_map.get(column.column_type, "")`